### PR TITLE
Fix disappearing cursor and Shift+Tab issue in Firefox

### LINKED ISSUE #90: Firefox selection and focus is broken
Whenever moving the focus to the toolbar with <kbd>Shift</kbd> + <kbd>Tab</kbd> on Firefox selection mechanism is broken right after that: the focus goes to the editor container and you cannot place the caret where you need it in the content.

Demo using the keyboard <kbd>arrows</kbd> and <kbd>Shift</kbd>, <kbd>Tab</kbd> buttons:

ff-bug

### PR DESCRIPTION
**Pull request by web-padawan**
Fixes #90

---
This change is [Reviewable](https://reviewable.io/reviews/vaadin/vaadin-rich-text-editor/93)

### CODE REVIEW

**Comment by web-padawan** [comment]
Updated the issue reproduction: https://jsfiddle.net/webpadawan/g6vku9L3/

**Comment by web-padawan** [comment]
Another issue in Firefox related to `overflow: hidden` https://jsfiddle.net/webpadawan/raox4mtu/14/

**Comment by web-padawan** [comment]
Related issues:

https://bugzilla.mozilla.org/show_bug.cgi?id=1496769 (existing)
https://bugzilla.mozilla.org/show_bug.cgi?id=1507101 (filed by me)

File: test/accessibility.html [hunk 1]
```diff
@@ -145,11 +145,21 @@
...
         });
 
-        it('should focus the editor on esc', done => {
+        it('should prevent keydown and focus the editor on esc', done => {
           sinon.stub(editor, 'focus', done);
           const e = new CustomEvent('keydown', {bubbles: true});
           e.keyCode = 27;
-          buttons[0].dispatchEvent(e);
+          const result = buttons[0].dispatchEvent(e);
+          expect(result).to.be.false;
```
Comment (yuriy-fix) [inline test/accessibility.html]: Nit: not sure, but probably a good idea to add a note that the return value is false if event is cancelable and some handler called `preventDefault()`.

File: test/accessibility.html [hunk 2]
```diff
@@ -134,9 +134,9 @@
           content.dispatchEvent(e);
         });
 
-        it('should mark toolbar as focused before blurring content on shift-tab', done => {
+        it('should mark toolbar as focused before focusin it on shift-tab', done => {
```
Comment (yuriy-fix) [inline test/accessibility.html]: focusin**g** ? By mistake was thinking about `focusin` event 😄

File: test/accessibility.html (hunk 1 above)
Comment (web-padawan) [reply test/accessibility.html]: Done.

File: test/accessibility.html (hunk 2 above)
Comment (web-padawan) [reply test/accessibility.html]: Done.

**Pull request merged**